EventListener/ExceptionListener.php

```diff
@@ -34,15 +35,25 @@ class ExceptionListener implements EventSubscriberInterface
 {
     private $exceptionListener;
     private $dispatcher;
+    private $innerExceptionListener;
 
-    public function __construct($controller, ?LoggerInterface $logger, EventDispatcherInterface $dispatcher)
+    public function __construct($controller, ?LoggerInterface $logger, $debug = false, EventDispatcherInterface $dispatcher, $innerExceptionListener = null)
```

Comment:
I wonder if with this approach we no longer need the `LegacyExceptionListener`, we could use the `ErrorListener` always, this way the exception attribute would be a \Throwable instance (by default) and we can get rid of the controller argument replacement logic as well.

Reply:
good point! will adjust this

Reply:
here is an example:  https://github.com/yceruto/FOSRestBundle/commit/3ae17f90bdbc01ef10f891a4a82e50ffaf3cc26a